### PR TITLE
Browser: Add window to inspect history

### PR DESCRIPTION
**Pull request by Sauler**
This PR adds ability to view history directly in browser :^)

image

**NOTE:** Currently it only shows history for current active tab.

### CODE REVIEW

**Comment by LucasChollet** [comment]
Sorry for the CI failure, it works again!

File: Userland/Applications/Browser/History/HistoryModel.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+DeprecatedString HistoryModel::column_name(int column) const
+{
+    switch (column) {
+    case Column::Title:
+        return "Title";
+    case Column::URL:
+        return "URL";
+    case Column::__Count:
+        return {};
```
Comment (MacDue) [inline Userland/Applications/Browser/History/HistoryModel.cpp:45]: Bit odd to handle the `__Count` case. Maybe just put `default: VERIFY_NOT_REACHED();` instead?

Comment (MacDue) [reply Userland/Applications/Browser/History/HistoryModel.cpp:45]: Then it's odd there too :^)

File: Userland/Applications/Browser/History.h [hunk 2]
```diff
@@ -18,6 +18,7 @@ class History {
         DeprecatedString title;
     };
     void dump() const;
+    Vector<URLTitlePair> get_all_history_entries();
```
Comment (linusg) [inline Userland/Applications/Browser/History.h:21]: why is this not just a getter for `m_items`?

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Applications/Browser/History/HistoryModel.cpp (hunk 1 above)
Comment (Sauler) [reply Userland/Applications/Browser/History/HistoryModel.cpp]: Fixed

File: Userland/Applications/Browser/History.h (hunk 2 above)
Comment (Sauler) [reply Userland/Applications/Browser/History.h:21]: Fixed

**Comment by Sauler** [comment]
Do i need to do something else?

**Review by ADKaster — APPROVED** [review]

**Pull request merged**